src/win/overlapped.c

```diff
@@ -0,0 +1,115 @@
+/* Copyright Joyent, Inc. and other Node contributors. All rights reserved.
```

Comment:
Quick note: use that copyright text: https://github.com/libuv/libuv/blob/1a96fe33343f82721ba8bc93adb5a67ddcf70ec4/src/win/snprintf.c#L1-L20